### DISCUSSION TITLE
how to add a Print icon on the top right corner of the Github page.

### DISCUSSION THREAD

**Discussion by lowrence-k1** [community/community#112569] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Query

### Body

Hi ,
how to add a Print icon on the top right corner of the Github page.
I use HUGO for UI
and Visual Studio Code for writing the code.

This print icon should be available Globally (that means, available on all the pages). User just has to click on the icon and be able to print that specific page.

Thanks. Waiting for a quick reply please...

**Comment by nadila2k** [comment]
To add a print icon on the top right corner of a GitHub page, you can utilize CSS and HTML. Here's a basic example:

Create an HTML file (index.html) for your GitHub page.
Add the following HTML and CSS:

**Comment by lowrence-k1** [comment]
Hi,

I am unable to view your reply.

can you please write back .

Thanks.

Regards,
Lowrence Kalyansingh
+91-9866-286-434
